### DISCUSSION TITLE
Next Js version conflict in Turborepo

### DISCUSSION THREAD

**Discussion by Nicolasvegam** [vercel/turborepo#6834] [Help]
When using the middleware of app Next js 13 instead of the Next js 14 (other app), typescript start to point at it as wrong.

<img width="1120" alt="image" src="https://github.com/vercel/turbo/assets/26209122/92b36015-13d5-4605-b0b0-041b2b880fd8">

Error message:
`La propiedad "[INTERNALS]" falta en el tipo "import("/Users/nicolas/Desktop/testing/carvuk-monorepo-clients/node_modules/.pnpm/next@13.4.12_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/web/spec-extension/request").NextRequest", pero es obligatoria en el tipo "import("/Users/nicolas/Desktop/testing/carvuk-monorepo-clients/node_modules/.pnpm/next@14.0.1_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/web/spec-extension/request").NextRequest".ts(2741)`

**Comment by anthonyshew** [comment]
Could you provide a minimal reproduction? 🙏

**Comment by anthonyshew** [comment]
Closing due to staleness.
